app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateWalletFragment.kt

```diff
@@ -89,19 +88,30 @@ class CreateWalletActivity : BaseActivity() {
     @JvmField
     var whiteColor = 0
 
-    override val contentViewId = R.layout.activity_create_wallet
-
     private val uiHandler = Handler()
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
+    override val contentViewId = R.layout.fragment_create_wallet
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
         init()
     }
 
+    private fun init() {
```

Comment:
Not sure if "init" is a good function name - it's reserved in Kotlin, "setupUI" may sound better. Not much happening here actually, you can even keep all this in onViewCreated.